v3/src/components/case-card/case-card.v2.test.tsx

```diff
@@ -7,6 +7,7 @@ import { createDataSet } from "../../models/data/data-set-conversion"
 import { DGDataContext } from "../../models/v2/dg-data-context"
 import { t } from "../../utilities/translation/translate"
 import "./case-card.v2"
+import { SharedCaseMetadata } from "../../models/shared/shared-case-metadata"
```

Comment:
These `metadata` imports and properties should be removed from the tests as well at some point.